tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java

```diff
@@ -3156,8 +3156,8 @@ public Set<String> getRawExtraPaths() {
         if (extraPaths == null) {
             extraPaths =
                     ImmutableSet.<String>builder()
-                            .addAll(getRawExtraPathsPrivate())
-                            .addAll(CONST_EXTRA_PATHS)
+                            .addAll(CharUtilities.internAll(getRawExtraPathsPrivate()))
```

Comment:
Could these be done with lambdas, eg getREPP().stream().map(x -> x.intern()).collect(...)

Reply:
It's an Iterable<> interface. Was trying to avoid having to allocate another copy of all of the strings. This way it's just pass through. But you could use a lambda on a temporary object

Reply:
PTAL